cmds/record.c

```diff
@@ -177,6 +178,15 @@ static void setup_child_environ(struct opts *opts, int argc, char *argv[])
 		}
 	}
 
+	if (opts->loc_filter) {
+		char *loc_str = uftrace_clear_kernel(opts->loc_filter);
+
+		if (loc_str) {
+			setenv("UFTRACE_LOCATION", loc_str, 1);
+			free(loc_str);
+		}
+	}
```

Comment:
What if user gave `-L` without `--srcline`?  Maybe we could set `--srcline` implicitly for record.

Reply:
@ekyooo You would need `setenv("UFTRACE_SRCLINE", "1", 1)` for this.

Please also specify this in `record` man page, but also specify in `live` page that `--srcline` is applied only for record not for replay.
```
Implies --srcline.
```
You can refer to https://github.com/namhyung/uftrace/blob/master/doc/uftrace-record.md#record-options.

Reply:
It's modified. Thank you.